src/ol_openedx_git_auto_export/test_settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for git_auto_export project.
+
+Generated by 'django-admin startproject' using Django 1.11.12.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+# pragma: allowlist nextline secret
+SECRET_KEY = "gq=bllmuiwwm9mq&hij(q)0#nbz*+km3ofnm_9_l@76cd2-869"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```

Comment:
This is the cause of a type-checking error in pre-commit. As it turns out we are not using it anywhere. As a matter of fact, we aren't using test_settings anywhere I believe. Not at least yet. But It'd make sense to delete at least this ALLOWED_HOSTS.

In addition, (https://github.com/mitodl/open-edx-plugins/issues/48)error is only raised in this PR because we only have this list here and it's not part of other plugins. As a reference look at [this canvas test_settings](https://github.com/mitodl/open-edx-plugins/blob/main/src/ol_openedx_canvas_integration/test_settings.py) it was generated through cookie-cutter so you can use the same i think.

Reply:
The main was just to copy that file and didn't really focused on anything. I did not fix any type checking error because it was not followed in any of the sibling plugins